Rubberduck.Parsing/VBA/ReferenceResolveRunnerBase.cs

```diff
@@ -110,7 +110,15 @@ private void ExecuteCompilationPasses(IReadOnlyCollection<QualifiedModuleName> m
                     new TypeHierarchyPass(_state.DeclarationFinder, new VBAExpressionParser()),
                     new TypeAnnotationPass(_state.DeclarationFinder, new VBAExpressionParser())
                 };
-            passes.ForEach(p => p.Execute(modules));
+            try
+            {
+                passes.ForEach(p => p.Execute(modules));
+            }
+            catch (Exception e)
+            {
+                _parserStateManager.SetModuleStates(modules, ParserState.ResolverError, token);
+                Console.WriteLine(e);
```

Comment:
Why write to the console here? Shouldn't we either log it or write to the the debug output, i.e. use `Debug.WriteLine`?